src/psyclone/tests/test_files/gocean1p0/kernel_invalid_declaration.f90

```diff
@@ -0,0 +1,42 @@
+module kernel_invalid_fortran
```

Comment:
Copyright/license please. Please also add a comment describing the purpose of the file.

Reply:
done.